dts/arm/nxp/nxp_rt1064.dtsi

```diff
@@ -17,5 +17,7 @@
 		spi-max-frequency = <133000000>;
 		status = "okay";
 		jedec-id = [ef 40 16];
+		erase-block-size = <4096>;
+		write-block-size = <4>;
```

Comment:
This doesn't look related to the commit message: `soc: nxp_imx: Disable IVT/DCD when building for mcuboot`

Reply:
True, but it is required if you want to sign the images for `mcuboot`.
Should I move this to another commit, or doesn't it belong to the PR at all?

Also the driver has this:
```C++
#define NOR_WRITE_SIZE  1
```

Which is hardcoded and doesn't match with what I added. Isn't the `dts` property preferred over the hardcoded value?

Reply:
Ah, I see. But why did you increase the write size to 4?

@galak how do you feel about this inheriting `soc-nv-flash.yaml` bindings?

Reply:
@galak ping